src/main/java/org/spongepowered/api/item/merchant/TradeOffer.java

```diff
@@ -128,6 +128,10 @@ static Builder builder() {
      */
     boolean doesGrantExperience();
 
+    float getPriceGrowthRate();
```

Comment:
Can this be a `double` instead? And can you javadoc it?

Reply:
Oh yeah forgot to mention that these are to compensate for the new properties of trade offers. Also thought it would be best not do have a 4 line PR just for that

Reply:
Need to java doc/link to mc wiki for information imo if you're doing that then.